packages/core/src/constants.ts

```diff
@@ -19,3 +20,54 @@ export const LOOKUP_DEFINITIONS_INDEX = 0;
 
 /** @internal */
 export const LOOKUP_DIR_INDEX = 1;
+
+/** @internal */
+export const UNSUPPORTED_CSS_PROPERTIES: Record<keyof CSS.StandardShorthandProperties, number> = {
```

Comment:
```suggestion
/** 
  * We use "1" as values to cause less effect on bundle size
  * @internal
  */
export const UNSUPPORTED_CSS_PROPERTIES: Record<keyof CSS.StandardShorthandProperties, 1> = {
```

Let's use `1` instead of `number` as we don't really want to allow anything except "1". And add a comment ;)